modular/manager/manager.go

```diff
@@ -479,7 +479,7 @@ func (m *ManageModular) LoadTaskFromDB() error {
 		}
 		replicateTask := &gfsptask.GfSpReplicatePieceTask{}
 		replicateTask.InitReplicatePieceTask(objectInfo, storageParams, m.baseApp.TaskPriority(replicateTask),
-			m.baseApp.TaskTimeout(replicateTask, objectInfo.GetPayloadSize()), m.baseApp.TaskMaxRetry(replicateTask))
+			m.baseApp.TaskTimeout(replicateTask, objectInfo.GetPayloadSize()), m.baseApp.TaskMaxRetry(replicateTask), meta.IsAgentUpload)
```

Comment:
check for if it agent upload object when loaded from DB, other wise the objectInfo from chain does not contain all info.

Reply:
the loaded task to seal is not working, please double check
`objectInfo, queryErr := m.baseApp.Consensus().QueryObjectInfoByID(context.Background(), util.Uint64ToString(meta.ObjectID))`
The objectInfo is queried from chain, how can it have missing checksums if its agent upload one?